### PR TITLE
Do not require space after case

### PR DESCRIPTION
**Pull request by simPod**
I think I missed this one. Currently I cannot write

```php
switch($y) {
    case 1:
    case 2:
        // do sth
}
```

but have to write

```php
switch($y) {
    case 1:

    case 2:
        // do sth
}
```

Which is IMO not desired. So this drops the requirement to have space after `case`.

### CODE REVIEW

File: tests/fixed/example-class.php [hunk 1]
```diff
@@ -129,6 +129,9 @@ public function trySwitchSpace() : void
         try {
             $var = 1;
             switch (self::VERSION) {
+                case 1:
+                case 2:
+                    // do something
```
Comment (carusogabriel) [inline tests/fixed/example-class.php]: Could you add a third `case` with a different action? Just to make sure that we're testing the expected :)

Comment (simPod) [reply tests/fixed/example-class.php]: Added, good?

**Comment by greg0ire** [comment]
@carusogabriel if that's the case the label should probably not be "improvement".

File: tests/input/example-class.php [hunk 2]
```diff
@@ -124,6 +124,13 @@ public function trySwitchSpace() : void
         try {
             $var = 1;
             switch (self::VERSION) {
+                case 1:
```
Comment (lcobucci) [inline tests/input/example-class.php]: Let's please isolate this in another file to make things easier to understand and maintain.

**Comment by simPod** [comment]
_@lcobucci is initializing_

I'll extract tests, wanted to do so for a long time already :)

File: tests/fixed/ControlStructures.php [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                // do something
+                break;
+            case 3:
+                // do something else
+                break;
+            default:
+        }
+    }
+
+    public function spaceBellowBlocks() : void
```
Comment (malarzm) [inline tests/fixed/ControlStructures.php]: Nitpicking, but it's "below" (one "l") :)

File: tests/input/ControlStructures.php [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+                // do something
+                break;
+            case 3:
+                // do something else
+                break;
+            default:
+        }
+    }
+
+    public function spaceBellowBlocks() : void
```
Comment (malarzm) [inline tests/input/ControlStructures.php]: same here

File: tests/fixed/ControlStructures.php (hunk 3 above)
Comment (simPod) [reply tests/fixed/ControlStructures.php]: daym, thought so but IDE spell check did not complain, it was wtf moment for me, did not validate it tho. Thanks.

**Comment by Ocramius** [comment]
Hmm, release `7.0.1` was botched due to this not targeting the `7.x` branch (`7.0.0` and `7.0.1` point to the same commit) : will need to move this to `7.0.2` and do some stuff manually when I got time

**Pull request merged**